archetypes/Pools/index.tsx

```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
+                <section className="mb-8">
+                    <h1 className="font-semibold text-3xl text-theme-text">Pools</h1>
+                    <FilterBar state={state} dispatch={dispatch} />
+                </section>
+                {!sortedFilteredTokens.length ? <Loading className="w-10 mx-auto mt-10" /> : null}
+                {Object.keys(groupedSortedFilteredTokens).map((key, index) => {
+                    const dataRows = groupedSortedFilteredTokens[key as any] as BrowseTableRowData[];
+                    return (
+                        <div
+                            key={index}
+                            className="p-4 md:p-8 lg:px-16 mb-4 shadow-xl rounded sm:rounded-2xl md:rounded-3xl bg-theme-background"
+                        >
+                            <div className="w-full h-20 my-5 p-4 flex justify-between divide-x-4 divide-cool-gray-200 dark:divide-cool-gray-900 overflow-x-auto whitespace-nowrap rounded-xl bg-cool-gray-50 dark:bg-theme-background-secondary">
+                                <div className="flex pr-10">
+                                    <Logo
+                                        className="inline mr-3 my-auto"
+                                        size="lg"
+                                        ticker={key.split('/')[0] as LogoTicker}
+                                    />
+                                    <div className="my-auto">
+                                        <div className="font-bold text-lg">{key}</div>
+                                    </div>
+                                </div>
+                                <div className="px-10">
+                                    <div className="text-cool-gray-500 dark:text-cool-gray-400 font-semibold">
+                                        SPOT PRICE
+                                    </div>
+                                    <div className="font-bold">{toApproxCurrency(dataRows[0].oraclePrice)}</div>
+                                </div>
+                                <div className="px-10">
+                                    <div className="text-cool-gray-500 dark:text-cool-gray-400 font-semibold">
+                                        ORACLE
+                                    </div>
+                                    <a
+                                        href={`https://reputation.link/contracts/${MarketToOracleMap(key)}`}
+                                        target="_blank"
+                                        rel="noreferrer"
+                                        className="flex items-center"
+                                    >
+                                        <img className="mr-2" src={'/img/general/chainlink.svg'} alt="Chainlink" />
+                                        <div className="mr-2 font-bold">Chainlink</div>
+                                        <img src={'/img/general/link.svg'} alt="Link" />
```

Comment:
If you import the image at the top it should change color when the font color changes
![image](https://user-images.githubusercontent.com/29347276/150242301-e9fd2847-44f8-4f4d-a11c-abd73c9c680d.png)

so do 
`import LinkIcon from 'whatever/path/to/img'`
and use it like
`<LinkIcon alt="Link"/>`

Reply:
✅